### PR TITLE
Add realtime best practices and link to validator

### PR DESCRIPTION
**Pull request by isabelle-dr**
A PR with a small change to the home page as I was reading through it:
- Adding the GTFS Realtime Best Practices
- Adding a link to the Validate page
- Adding a small paragraph about data accuracy

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -39,12 +39,16 @@ GTFS data is now being used by a variety of software applications for many diffe
...
+For information on creating high quality data, see the [California Transit Data Guidelines](https://dot.ca.gov/cal-itp/california-transit-data-guidelines), the [GTFS Schedule Best Practices](https://gtfs.org/schedule/best-practices/) and the [GTFS Realtime Best Practices](https://gtfs.org/realtime/best-practices/).
+For evaluating the quality of a dataset, see [Validate](https://gtfs.org/schedule/validate/) page.
 
 ### Complete Data
 
 Quality GTFS includes important service details such as holiday and summer schedule changes, accurate stop locations, and names for routes and headsigns that match other rider-facing materials. Even if an agency works with a vendor to produce GTFS, it’s ultimately up to the agency to ensure that the information presented in print, on board, and online is consistent.
 
-For information on creating high quality data, see the [California Transit Data Guidelines](https://dot.ca.gov/cal-itp/california-transit-data-guidelines) and the [GTFS Best Practices](https://gtfs.org/schedule/best-practices/)
+### Accurate
```
Comment (tzujenchanmbd) [inline docs/index.md]: Suggest to be "Accurate Data" to maintain consistency with the above "Complete Data"

Comment (tzujenchanmbd) [reply docs/index.md]: https://github.com/MobilityData/gtfs.org/pull/246/commits/1c0fa0cf476893bdf69ca2c6e0204b21398b03e7

**Review by tzujenchanmbd — APPROVED** [review]
LGTM, 
1 tiny editing change

**Comment by isabelle-dr** [comment]
Thanks @tzujenchanmbd for the review and fix! Merging this PR, I've added it to the translation backlog. 
Screenshot 2023-12-08 at 3 47 11 PM

**Pull request merged**